modules/reliability/php/NDB_Menu_Filter_reliability.class.inc

```diff
@@ -507,11 +444,43 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
         $DB->insert('reliability',array('CommentID'=>$CommentID,
                'reliability_center_id'=>$params['CenterID'],
                'Instrument'=>$params['Instrument'],
-               'Manual_Swap'=>'no') 
+               'Manual_Swap'=>'no')
         );
         return array("message" => "Inserted new reliability candidate".$params['PSCID']."/".$params['Visit_label'] );
     }
-    
+
+    /**
+     * Override toJSON to append a calculated field to the standard output
```

Comment:
I find the usage of the phrase "standard output" a little confusing, since it kind of sounds like a reference to STDOUT.